CMakeLists.txt

```diff
@@ -178,7 +178,7 @@ option(BUILD_UTILITIES "Build stand-alone executables for utilities that can als
 option(NEW_PARSER_DEBUG "Enable tracing of new parser" OFF)
 option(BUILD_MULTI_CORE "Enable building for multicore system" ON)
 option(FAIL_MISSING "Fail when a required external dependency is not present (useful for packagers)" OFF)
-option(USE_GETTEXT "Use the Gettext internationalization library" ON)
+option(USE_GETTEXT "Use the Gettext internationalization library" OFF)
```

Comment:
Not sure this should be set to OFF by default. It should gracefully not use gettext if it's not found.

Reply:
@vlazzarini can you revert this change for this PR as it doesn't have to do with fixing the tests? We can track it in the other PR.

Reply:
done